### DISCUSSION TITLE
Increasing the amout of Good Inbound Peers:

### DISCUSSION THREAD

**Discussion by BonzoDogDooDah** [lightningnetwork/lnd#8742] [Q&A]
Hi, On  our node which is named: Handshake our rating is 1 good Inboud Peer to 10 good outbound peers. We wish to have a lot more good inbound peers so as to increase our routing flow, etc... Can you please tell us how we can achieve this, how to we get more than 1 good inbound peer because as we all know how important it is to have good inbound peers because without them nothing will flow in and then obviously out of the node.  Thank you :)

**Comment by bitromortac** [comment]
I think the easiest way would be to open up channels to nodes on the list, maybe 2-5 Msat in channel size (or more) and then loop out part of the channel (other ways would be to buy channels on pool, or to ask node operators to open channels to you). You already have connections to popular nodes, often it is better to also connect to smaller nodes that don't have a very good connection to the network, those channels add value. All first ~1000 nodes on the list can in principle become good inbound peers. Just a heads-up that the newly created peers won't show up immediately as good inbound peers, that will take a bit of time.

**Comment by BonzoDogDooDah** [comment]
Hi bitromortac,

Thank you for this information we will use it wisely. :)

BonzoDogDooDah

-- 
 Sent with Tuta; enjoy secure & ad-free emails: 
 https://tuta.com


May 9, 2024, 06:23 by ***@***.***:
